Tests/GRPCCoreTests/MetadataTests.swift

```diff
@@ -17,6 +17,19 @@ import GRPCCore
 import XCTest
 
 final class MetadataTests: XCTestCase {
+  func testInitFrom() {
```

Comment:
Let's be a touch more specific: 
```suggestion
  func testInitFromSequence() {
```